src/app/containers/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from "react";
+import {
+  Button,
+  CardActions,
+  CardContent,
+  Typography,
+  Grid,
+} from "@material-ui/core";
+
+import { LogoutUser, GetNewToken } from "../../../auth/core/redux/actions";
+
+import { connect } from "react-redux";
+
+import { useDispatch } from "react-redux";
+
+import { Card, CardHeader } from "@material-ui/core";
+
+import { acceptPrivacyPolicy } from "../../../auth/core/services/privacypolicy.service";
+
+function Dashboard(props: any) {
+  const dispatch = useDispatch();
+  const [disabled, setDisabled] = useState(false);
```

Comment:
Not sure what is disabled. Can we be more explicit here?

Reply:
Made changes.